src/components/companies/users/form.vue

```diff
@@ -110,6 +110,23 @@
                                 />
                                 <span class="text-danger">{{ errors.first("role") }}</span>
                             </div>
+
+                            <div class="form-group">
+                                <label> Status </label>
+                                <div>
+                                    <label for="user_active">
```

Comment:
```suggestion
                                    <label>
```

`for` is not necessary if the `input` is inside the `label`.